src/Review/Rule.elm

```diff
@@ -2642,8 +2642,8 @@ withDeclarationExitVisitor visitor (ModuleRuleSchema schema) =
 
 {-| Add a visitor to the [`ModuleRuleSchema`](#ModuleRuleSchema) which will visit the module's
 [declaration statements](https://package.elm-lang.org/packages/stil4m/elm-syntax/7.2.1/Elm-Syntax-Declaration)
-(`someVar = add 1 2`, `type Bool = True | False`, `port output : Json.Encode.Value -> Cmd msg`),
-collect data and/or report patterns.
+(`someVar = add 1 2`, `type Bool = True | False`, `port output : Json.Encode.Value -> Cmd msg`)
+in source order to collect data and/or report patterns.
```

Comment:
I think it would be clearer something like this:
```suggestion
(`someVar = add 1 2`, `type Bool = True | False`, `port output : Json.Encode.Value -> Cmd msg`),
to collect data and/or report patterns. The declarations are in the same order that they appear in in the source code.
```

Reply:
Makes sense to me; I pushed a commit accordingly (there was a typo here so didn't want to commit the suggestion).